frontend/components/site/siteSettings.js

```diff
@@ -112,61 +112,53 @@ class SiteSettings extends React.Component {
         </div>
         <div className="usa-grid">
           <div className="usa-width-one-whole">
-            <div className="form-group">
-              <label className="active" htmlFor="domain">Custom domain</label>
-              <input
-                name="domain"
-                className="form-control"
-                type="text"
-                placeholder="https://example.com"
-                value={ state.domain }
-                onChange={this.onChange} />
-            </div>
-            <div className="usa-alert usa-alert-info">
-              <div className="usa-alert-body">
-                <h3 className="usa-alert-heading">Custom Domain</h3>
-                <p className="usa-alert-text">This read-only origin box is used for Federalist to set a custom domain on the 18F side. Federalist customers only need this information if handling custom URL work directly.</p>
-                <input readOnly type="text" value="" />
+            <div className="form-group">              
+              <div className="usa-alert usa-alert-info">
+                <div className="usa-alert-body">
+                  <h3 className="usa-alert-heading">Custom Domain</h3>
+                  <p className="usa-alert-text">If you build your site with Jekyll, Federalist can configure your production site to load at a custom domain specified here.</p>
+                  <input
+                   name="custom-domain"
```

Comment:
The name here needs to be `domain` instead of `custom-domain`, or the JS won't be able to find this text field when it updates.